src/course/syllabus/application/pet-shop/project.md

```diff
@@ -0,0 +1,58 @@
+For this project, we're giving you some data in a JavaScript object. Create an interface where the user can interact with and change what is displayed on your webpage. Interacting with page will alter and update the data stored in the object.
+
+What you build and how you interpret the data is up to you. We are looking to see your own way of displaying and interacting with the data.
+
+## Requirements
+
+- [ ] Store the object below in a JavaScript file in your codebase
+- [ ] Render the data into HTML using DOM Manipulation
+- [ ] Allow the user to update object values when interacting with the webpage
+
+## Data
+
+Your data is stored in the following object:
+
+```
+let dataObject = {
+  1: {
```

Comment:
I'm not sure an object with number keys is a great idea for a beginner thing. This is effectively recreating an array, but without all the convenient methods arrays normally have.

If each dog has a unique name I would recommend using that as the key instead, as it's a more realistic example of using objects. E.g.

```js
let dataObject = {
  fido: {
    name: "fido",
    type: "dog",
  }
}
```
Up to you whether to repeat the name again in the object—it can make using the data easier, but it is duplication.

Reply:
Yeah, I'm not sure here what the best approach is. The most natural feels like it would be to include an array of objects. Or, as you say, using unique keys for each pet. Do you have an opinion on which is better practice?

Reply:
Perhaps a simpler object is a better approach, for example:

```js
let user = {
  name: "Gregor",
  github: "Albadylic",
  age: 28,
  hobbies: ["pickling", "fermenting", "making kombucha", "cooking", "playing Overcooked"]
}